### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by snigehere** [wills106/homeassistant-solax-modbus#533] [Q&A]
Hi - I have a solis inverter (first 6 of the serial are 163AF1) and am able to get data via the cloud but wanted to switch to local and was pointed to this integration.

My inverter wifi data logger is on address 192.168.0.249 and my home assistant on 192.168.0.239

<html>
<body>

Version | core-2023.9.2
-- | --
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.11.5
Operating System Family | Linux
Operating System Version | 6.1.45
CPU Architecture | x86_64
Timezone | Europe/London
Configuration Directory | /config

</body>
</html>

I have a wifi data logger (firmware version MW_08_512_0501_1.94) and the mobus server is set up (from config_hide) .. It seems that the address of the server is not configurable and shows 10.10.100.254

![Snap2](https://github.com/wills106/homeassistant-solax-modbus/assets/104912027/f4c37f6b-f40d-4e0c-9218-6fda3b35c8c9)

I have installed the integration and think I have correctly configured but not sure

The prefix to be used for your inverter sensors : solis_mobus 
Interface: TCP / Ethernet
The modbus address of the Inverter: 1
Select Inverter Type: solis (I have tried both solis and solis_old as the inverter)
The polling frequency of the modbus registers in seconds: 15
Emergency Power Option (EPS): not selected
Dry Contact Box (Gen4): not selected
Parallel Mode (Master-Slave): not selected

TCP/IP Parameters
The IP-address of your Inverter or Modbus Interface: 192.168.0.249
The TCP port on which to connect to the inverter: 8899

Modbus TCP: selected
Modbus RTU over TCP: not selected
Modbus ASCII over TCP: not selected

When I kick off the integration I get solis_mobus: reading serial number from address 0xbf5 failed; other address may succeed

I get the understanding that this means nothing came back from the inverter but not sure 

Any pointers or help would be appreciated.

**Comment by wills106** [comment]
You'll need to use Integration 2023.09.4 as there was a breaking change in HA 2023.9.x
There is a link to some of the known working/not-working Solis Data loggers here. https://github.com/wills106/homeassistant-solax-modbus/discussions/521#discussioncomment-6946421 Your Inverter isn't actually supported yet, but you don't have a working Modbus connection yet. When you have a working connection it will output your Serial as an error.

**Reply by snigehere** [reply]
Thanks for the fast response.  I checked I have 2023.09.4 installed.. just to be sure I re downloaded it and restarted.. same results so am suspecting the data logger is not working well.  I tried to use server b previously and that didn't work either.  I am going to see if I can get a DLS-L stick and see if that helps.

I have had my inverter about 18 months and been looking on and off for a local solution.. I have tried a couple of methods without success.  I have had a lengthy back and forth with solis support and not made any progress other than to find my logger does not support server b.  They have set my logger to 1 minute cycle but looking at the actual data I get into HA via the API it's timestamp varies..sometimes 60 seconds old, sometimes 5 minutes old so its not ideal hence looking for a local solution again.
